examples/GPU/example_3d_trajectory_display.py

```diff
@@ -0,0 +1,117 @@
+Another key feature is to display the sampling density in k-space, for example to check for k-space holes or irregularities in the learning-based trajectories that would lead to artifacts in the images.
+"""
+
+# %%
+# Imports
+from mrinufft.trajectories.display3D import get_gridded_trajectory
+import mrinufft.trajectories.trajectory3D as mtt
+from mrinufft.trajectories.utils import Gammas
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# %%
+# Utility function to plot mid-plane slices for 3D volumes
+def plot_slices(axs, volume, title=""):
+    def set_labels(ax, axis_num=None):
+        ax.set_xticks([0, 32, 64])
+        ax.set_yticks([0, 32, 64])
+        ax.set_xticklabels([r"$-\pi$", 0, r"$\pi$"])
+        ax.set_yticklabels([r"$-\pi$", 0, r"$\pi$"])
+        if axis_num is not None:
+            ax.set_xlabel(r"$k_" + "zxy"[axis_num] + r"$")
+            ax.set_ylabel(r"$k_" + "yzx"[axis_num] + r"$")
+
+    for i in range(3):
+        volume = np.rollaxis(volume, i, 0)
+        axs[i].imshow(volume[volume.shape[0] // 2])
+        axs[i].set_title(
+            ((title + f"\n") if i == 0 else "") + r"$k_{" + "xyz"[i] + r"}=0$"
+        )
+        set_labels(axs[i], i)
+
+
+# %%
+# Helper function to showcase the features of `get_gridded_trajectory` function
+# This function will first grid the trajectory using the `get_gridded_trajectory`
+# function and then plot the mid-plane slices of the gridded trajectory.
+def create_grid(grid_type, trajectories, traj_params, **kwargs):
+    fig, axs = plt.subplots(3, 3, figsize=(10, 10))
+    for i, (name, traj) in enumerate(trajectories.items()):
+        grid = get_gridded_trajectory(
+            traj,
+            traj_params["img_size"],
+            grid_type=grid_type,
+            traj_params=traj_params,
+            **kwargs,
+        )
+        plot_slices(axs[:, i], grid, title=name)
+    plt.tight_layout()
+
+
+# %%
+# Create a bunch of sample trajectories
+trajectories = {
+    "Radial": mtt.initialize_3D_phyllotaxis_radial(64 * 8, 64),
+    "FLORETs": mtt.initialize_3D_floret(64 * 8, 64, nb_revolutions=2),
+    "Seiffert Spirals": mtt.initialize_3D_seiffert_spiral(64 * 8, 64),
+}
+traj_params = {
+    "FOV": (0.23, 0.23, 0.23),
+    "img_size": (64, 64, 64),
+    "gamma": Gammas.HYDROGEN,
+}
+
+# %%
+# Display the density of the trajectories, along the 3 mid-planes. For this, make `grid_type="density"`.
+create_grid("density", trajectories, traj_params)
+plt.suptitle("Sampling Density")
+plt.show()
+
+
+# %%
+# Display the sampling times over the trajectories. For this, make `grid_type="time"`.
+# It helps to check the sampling times over the k-space trajectories, which can be responsible for excessive off-resonance artifacts.
+# Note that this is just a relative visualization of sample times on a colour scale, and the actual sampling time.
```

Comment:
In the docs this piece of text appears closer to the image above than the one below so it's not clear which one it talks about when we are in the middle of the page.
I would suggest to create sections to make it visually clear, and also that way we have an automatic summary of the whole example which is always nice

Reply:
I think this is handeled now.